### PR TITLE
v-for should have key

### PR DESCRIPTION
**Pull request by ChangJoo-Park**

### CODE REVIEW

**Comment by cristijora** [comment]
@ChangJoo-Park Thanks for the PR. Indeed it points out to an issue/warning, but I would use something else than the index if possible. Maybe `link.name + index`

File: src/components/UIComponents/SidebarPlugin/SideBar.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
       </slot>
       <ul :class="navClasses">
         <!--By default vue-router adds an active class to each route link. This way the links are colored when clicked-->
-        <router-link v-for="(link,index) in sidebarLinks" :to="link.path" tag="li" :ref="link.name">
+        <router-link v-for="(link,index) in sidebarLinks" :to="link.path" tag="li" :ref="link.name" :key="index">
```
Comment (cristijora) [inline src/components/UIComponents/SidebarPlugin/SideBar.vue]: Would be nice to set the `:key="link.name+index"`

Comment (ChangJoo-Park) [reply src/components/UIComponents/SidebarPlugin/SideBar.vue]: yes I agree :) I'll update my PR

Comment (mubaidr) [reply src/components/UIComponents/SidebarPlugin/SideBar.vue]: Is there any reason you prefer to use combination of name and index? In my opinion its just an overhead.

Comment (cristijora) [reply src/components/UIComponents/SidebarPlugin/SideBar.vue]: @mubaidr https://vuejs.org/v2/guide/list.html#key

Doc quote 

This default mode is efficient, but only suitable when your list render output does not rely on child component state or temporary DOM state (e.g. form input values).

To give Vue a hint so that it can track each node’s identity, and thus reuse and reorder existing elements, you need to provide a unique key attribute for each item. An ideal value for key would be the unique id of each item.

To summarize: 
Index is good as a key but if possible, something more than an index should be added as a key.

**Comment by ChangJoo-Park** [comment]
Thanks 👍
